elections/steering/2025/README.md

```diff
@@ -136,22 +136,20 @@ Examples of contributions that would NOT be considered:
   public announcement is 24-48 hours.
 -->
 
-**The schedule below is an example and has not been updated for 2025**
-
 | Date                    | Event                                                                 |
 |:------------------------|:----------------------------------------------------------------------|
-| Tuesday, July 9         | Steering Committee selects Election Committee                         |
-| Thursday, August 1      | Announcement of Election and publication of voters.md                 |
-| Wednesday, August 21    | Steering Committee Q+A for the candidates (to be confirmed)           |
-| Saturday, August 24     | Candidate nominations due at the end of the day in AoE time           |
-| Sunday, August 25       | All candidate bios due at the end of the day in AoE time              |
-| Tuesday, August 27      | Election Begins                                                       |
-| Monday, September 23    | Deadline to submit voter exception requests                           |
-| Thursday, September 26  | Election Closes at the end of the day in AoE time                     |
-| Friday, September 27    | Private announcement of Results to SC members not up for election     |
-| Monday, September 30    | Private announcement of Results to all candidates                     |
-| Wednesday, October 2    | Public announcement of Results at Public Steering Committee Meeting   |
-| Wednesday, October 9    | Election Retro                                                        |
+| Wednesday, July 23      | Steering Committee selects Election Committee                         |
+| Monday, August 25       | Announcement of Election and publication of voters.md                 |
+| Thursday, September 4   | Steering Committee Q+A for the candidates (to be confirmed)           |
```

Comment:
[I might be out for a bit following the release, but there are 7 of us, and I'm happy to answer questions async.]

Reply:
I like the idea of collapsing events if possible